packages/theme-tasks/lib/tasks/sass.js

```diff
@@ -41,6 +41,8 @@ const dartSassOptions = {
 
 // #region core
 function build(fileGlob = paths.sass.src, options = quickSassOptions) {
+    options.importer.resetImported();
```

Comment:
I believe this should not be included in the commit

Reply:
After rebase, the commit will be gone